shell/platform/android/io/flutter/embedding/engine/plugins/util/GeneratedPluginRegister.java

```diff
@@ -33,11 +33,15 @@ public static void registerGeneratedPlugins(@NonNull FlutterEngine flutterEngine
           generatedPluginRegistrant.getDeclaredMethod("registerWith", FlutterEngine.class);
       registrationMethod.invoke(null, flutterEngine);
     } catch (Exception e) {
-      Log.w(
+      Log.e(
           TAG,
           "Tried to automatically register plugins with FlutterEngine ("
               + flutterEngine
               + ") but could not find and invoke the GeneratedPluginRegistrant.");
+      Log.e(
+        TAG,
+        // getCause here because the first layer of the exception would be from reflect.
+        "Received exception while registering: " + e.getCause());
```

Comment:
It would be nice if we could print out the name of the plugin that created the problem too.

Reply:
Ya, I'd like to buff up the actual content of the method being called by reflection too. Though that's in the flutter/flutter repo (that Java file is made by flutter_tools). Right now, I think it does state which plugin caused the problem (in the exception call stack which has the plugin's package name in it).